README.md

```diff
@@ -53,6 +53,9 @@ generic with minimal need for project-specific behavior.
 
 - AWS credentials configured in `~/.aws/credentials` and/or `~/.aws/config`
 
+- macOS Big Sur (11.x) users should follow additional steps mentioned 
+  in [Troubleshooting](#installing-python-383-on-macos-bigsur)
```

Comment:
```suggestion
- User of macOS Big Sur (11.x) should follow additional steps mentioned 
  in [Troubleshooting](#installing-python-383-on-macos-bigsur)
```

Reply:
```suggestion
- macOS 11 (Big Sur) or later users should follow additional steps mentioned 
  in [Troubleshooting](#installing-python-383-on-macos-bigsur)
```

And in kind everywhere else. In general, I want to reduce the amount that the code name is used, in favor of the version number. But most importantly we should account for the fact that the same fixes may be necessary in subsequent versions.